php/FlatbufferBuilder.php

```diff
@@ -593,6 +593,10 @@ public function endVector()
 
     protected function is_utf8($bytes)
     {
+        if (function_exists('mb_detect_encoding') && (bool) mb_detect_encoding($bytes, 'UTF-8', true)) {
```

Comment:
Yes, you're right, I really did a bad job there, just editing the file on github directly. Sorry for your time.

I'll checkout the project, do that change properly and add a new test, but I'm not sure how to test both behaviors, one using mb_detect_encoding and the other using the hexadecimal algorithm.